src/modules/auth/actions/set-login-account.js

```diff
@@ -1,12 +1,12 @@
 import { useUnlockedAccount } from 'modules/auth/actions/use-unlocked-account'
-import { loadAccountTrades } from 'modules/my-positions/actions/load-account-trades'
+import { loadAccountHistory } from 'modules/auth/actions/load-account-history'
 import logError from 'utils/log-error'
 
 // If there is an available unlocked account, set as the user's sending address.
 export const setLoginAccount = (autoLogin, account, callback = logError) => (dispatch, getState) => {
   // If autoLogin=true, use an unlocked local Ethereum node (if present)
   if (autoLogin) {
     dispatch(useUnlockedAccount(account, callback))
-    dispatch(loadAccountTrades())
+    dispatch(loadAccountHistory())
```

Comment:
I think it would make more sense for `loadAccountHistory` to be called at the end of `useUnlockedAccount`.  If the account isn't actually unlocked, we don't want to load the history.